2024/2024-ssl-teams-vs-teams-passing-challenge.adoc

```diff
@@ -0,0 +1,105 @@
+
+Therefore this year two teams will try to play as many passes as possible while competing against each other. To achieve this, the ruleset used last year has been adapted to the new situation.
+
+Although the teams, which have participated in one of the previous passing challenges, might have a slight advantage, we would like to encourage all teams to participate in this technicall challenge.
+
+== Procedure
+
+The technicall challenge will be held on a Div B field with a maximum of 4 robots per team.
+
+To level the playing field and to remove the impact of the performance of the opponent team on a teams score, each team will play two games against different contestants of the technical challenge.
+
+Each game will have the following procedure:
+
+* If both teams agree on the team colors, their suggestion will be used. If they do not agree, the referee will make a coin toss and the winning team will get the yellow team color. Even in this case, the teamcolor will not be changed during the half time break.
+* To start the game, a kick-off command for the yellow team will be issued from the game controller. For the start the robots can be positioned can positioned in any way prefered, along as the positioning does not violate the usual ruleset for a kick-off.
+* After *2 minutes of game time* a stop command will be issued and the first half of the game is over.
+* The half time break for this challenge will be only 60 seconds, since it should be mainly used to setup the robots at their desired position again and to exchange faulty robots. 
+* To start the second half a kick-off command for the blue team will be issued.
+* After another *2 minutes of game time* a stop command will be issued and the game is over. 
+
+
+=== Pass Definitions
+
+Teams are free in how they perform the passes. Definitions of a pass in general and additional constrains for valid pass that are considered in the scoring are given in this section.
+
+A pass is defined as follows:
+
+* A pass has a source position, a target position and a shooter.
+* The source position is where a robot kicks the ball.
+* The target position is where the ball stops or changes direction by more than *30 degrees* in field plane (x, y).
+* The shooter is the robot located nearest to the source position when the pass starts.
+* The pass distance is the euclidean distance between the source and target position.
+* The pass direction is the orientation of the `target - source` vector.
+* The pass is considered *chipped*, if the ball reached a maximum height of at least *15cm*.
+
+=== Special Ruleset during the technical challenge
+
+For the technical challenge a Div B field will be used. If the ball leaves the field via any of the field lines, the game will be halted and the team, that did not last touch the ball gets awared a free kick.
```

Comment:
why halted? Can't we just apply normal rules here? Stop the game, place the ball (by human?) and continue with free kick, as defined by the match rules?

Reply:
If we decide to go with a free kick, it could definitely be simplified. But I am currently still thinking about whether moving the ball further to the middle of the field, or starting with a kick-off after a ball left the field would be a better idea.
My reasoning behind moving the ball more to the middle would be to reduce the chance of it getting out of bounds again, but I am not sure, if that is that effective for the added time it could require.
But Free Kick vs. Kick-Off would be one of the things, it like to discuss in the next meeting to hear some other thoughts about it.